build/steps-build-ios.yml

```diff
@@ -58,6 +61,16 @@ steps:
     bundleIdentifier: $(ApplicationIdentifier)
     printFile: true
 
+- task: LaunchIconBadge@1 
+  condition: ne(variables['ApplicationEnvironment'], 'Production')
```

Comment:
I think it would make more sense if we check whether we're really set `BannerVersionNameColor`. Especially on project where we would want to push a prod build in internal (for project that has actual differences in between those builds)

Reply:
I propose that we do not display a banner for prod builds